tests/deploy.test.js

```diff
@@ -20,65 +20,24 @@ describe('Deploying a CF app to IBM Cloud', () => {
   it('runs the right set of commands', async () => {
     getInput.mockImplementation((name) => ({ 'cf-app': 'cf-app-foo' }[name]));
     await deploy();
-    expect(exec).toHaveBeenNthCalledWith(
-      1,
-      'ibmcloud',
-      ['cf', 'add-plugin-repo', 'CF-Community', 'https://plugins.cloudfoundry.org'],
-      execOptions
-    );
-    expect(exec).toHaveBeenNthCalledWith(
-      2,
-      'ibmcloud',
-      ['cf', 'install-plugin', 'blue-green-deploy', '-f', '-r', 'CF-Community'],
-      execOptions
-    );
-    expect(exec).toHaveBeenNthCalledWith(
-      3,
-      'ibmcloud',
-      ['cf', 'blue-green-deploy', 'cf-app-foo', '--delete-old-apps'],
-      execOptions
-    );
+    expect(exec).toHaveBeenNthCalledWith(3, 'ibmcloud', ['cf', 'push', 'cf-app-foo', '--strategy', 'rolling'], execOptions);
```

Comment:
The 1st arg of `toHaveBeenNthCalledWith()` means "the nth call of the spy". Probably it explains the CI failure? I see two other occurrences of the same pattern.

```suggestion
    expect(exec).toHaveBeenNthCalledWith(3, 'ibmcloud', ['cf', 'push', 'cf-app-foo', '--strategy', 'rolling'], execOptions);
```

Reply:
Yeah, thanks! 🎉